src/routes/taxIdentifications.ts

```diff
@@ -7,9 +7,23 @@ import {
 } from "../db";
 
 import { createChangeRequest } from "./changeRequest";
+import { isSteuerIdValid } from "validate-steuerid";
 
 export const TIN_UPDATE = "Patch/tax-indentifications/id";
 
+const deTaxIdValidationError = {
```

Comment:
This creates only one id as far as i understand, if we want separate ids, we need to wrap this in a function

Reply:
@DanaNussair How do we use this ID field? If it's not useful we can also remove it.

Reply:
I guess because it is part of the schema https://docs.solarisgroup.com/guides/get-started/digital-banking/onboard-person/#tax-id-validation